histomicstk/cli/ColorDeconvolution/ColorDeconvolution.py

```diff
@@ -12,20 +13,30 @@
 logging.basicConfig()
 
 
-def main(args):
-    import skimage.io
-
-    # Read Input Image
-    print('>> Reading input image')
-
-    print(args.inputImageFile)
-
+def colorDeconvolve(args):
+    sink0 = large_image.new()
+    sink1 = large_image.new()
```

Comment:
if `args.stain_2 == 'null'`, we shouldn't be creating sink1, and likewise with `args.stain_3` and sink2.  I realize we were creating pngs before, but not adding them as annotations.